packages/fab/index.js

```diff
@@ -22,13 +22,15 @@ export class Fab extends React.Component {
     });
   }
 
-  addIconClassToAllChildren = () => {
-    return React.Children.map(this.props.children, (item) => {
-      const className = `${item.props.className} mdc-fab__icon`;
-      const props = Object.assign({}, item.props, {className});
-      return React.cloneElement(item, props);
-    });
-  };
+  renderIcon() {
+    const {icon} = this.props;
+
+    if (!icon) {
+      return;
+    }
+
+    return this.addClassesToElement('mdc-fab__icon', icon);
```

Comment:
since you only use this `addClassesToElement` once, I think you should inline the logic:

```
const updatedProps = {
      className: classnames('mdc-fab__icon', icon.props.className),
    };

    return React.cloneElement(icon, updatedProps);
```